.circleci/build_prod_config.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import json
+
+with open('../dist/js-hash.json', 'r') as js_hash_file, open('../dist/css-hash.json', 'r') as css_hash_file:
+    js_hash = json.load(js_hash_file)
+    css_hash = json.load(css_hash_file)
```

Comment:
Minimise the time you have the files open and simplify this code at the same time:

```python
js_hash = json.load(open('../dist/js-hash.json'))
css_hash = json.load(open('../dist/css-hash.json'))
```

or if you prefer using the context manager (`with` statement):

```python
with open('../dist/js-hash.json') as js_hash_file:  # the 'r' is default and usually omitted
    js_hash = json.load(js_hash_file)

with open('../dist/css-hash.json') as css_hash_file:
    css_hash = json.load(css_hash_file)
```

but don’t put everything in one giant `with` block.